tests/view_tests/tests/test_debug.py

```diff
@@ -263,6 +263,38 @@ def test_template_exceptions(self):
                     "traceback, instead found: %s" % raising_loc
                 )
 
+    def test_safestring_in_exception(self):
+        with self.assertLogs('django.request', 'ERROR'):
+            response = self.client.get("/safestring_exception/")
+            # HTML is specifically turned off so that unbalanced tags (to break
+            # out of extant HTML) are OK.
+            self.assertNotContains(
+                response,
+                '<h3></textarea><script>alert(1);</script></h3>',
```

Comment:
`</textarea><script>alert(1);</script>` appears in multiple places on a debug page, e.g.

![image](https://user-images.githubusercontent.com/2865885/151308176-80843436-86ee-4abb-ab37-fb86e777b47e.png)
![image](https://user-images.githubusercontent.com/2865885/151308204-103f7498-4f84-4a57-81ee-627a7535bff9.png)
![image](https://user-images.githubusercontent.com/2865885/151308332-92977d52-98f3-4729-98a1-f95356b21122.png)

so I'd remove `h3` tags from assertions to cover all places.

Reply:
Decent catch, I noticed after writing the first assertion that it appeared and could be broken out of twice (second time in the textarea), and just popped another one on additionally, will fix. Thanks.